applications/HDF5Application/custom_io/hdf5_file.cpp

```diff
@@ -22,67 +49,9 @@ bool IsPath(const std::string& rPath)
     return regex_match(rPath, std::regex("(/[\\w\\(\\)]+)+"));
```

Comment:
perhaps this can be done better with `std::filesystem`

Reply:
I am not sure how to do this with `std::filesystem` since these paths are not actual file paths, but hdf5 paths within one hdf5 file.